### PR TITLE
Add End Bricks & End Brick Stairs

### PR DESCRIPTION
**Pull request by Aericio**
# Blocks

* End Bricks
* End Brick Stairs

Should I create a separate PR to add end brick slabs / end brick walls, since they are grouped with other slabs/walls? or just add to here.

### CODE REVIEW

**Comment by Aericio** [comment]
> ##[error]dragonfly/block/end_brick_stairs.go:59:10: undefined: noCorner
##[error]dragonfly/block/end_brick_stairs.go:59:27: undefined: cornerRightInner
##[error]dragonfly/block/end_brick_stairs.go:59:52: undefined: cornerRightOuter
##[error]dragonfly/block/end_brick_stairs.go:65:10: undefined: cornerRightOuter
##[error]dragonfly/block/end_brick_stairs.go:69:17: undefined: cornerLeftOuter
##[error]dragonfly/block/end_brick_stairs.go:73:17: undefined: cornerRightInner
##[error]dragonfly/block/end_brick_stairs.go:77:17: undefined: cornerLeftInner
##[error]dragonfly/block/end_brick_stairs.go:119:10: undefined: cornerRightOuter

GoLand wanted me to remove those tho...

**Comment by Sandertv** [comment]
Make sure to get this up to date with the latest changes :smile: If you have a look at wood stairs, you'll see what needs to happen. All the AABB stuff now relies on the model.

File: dragonfly/block/end_bricks.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item"
+
+// EndBricks is a block made from combining four endstone blocks together
```
Comment (Sandertv) [inline dragonfly/block/end_bricks.go:5]: Make sure to add a `.` here.

**Comment by Sandertv** [comment]
As for slabs and walls, please feel free to put them in a separate PR. Walls are currently not implemented at all so they'll require some more work.

**Pull request merged**